ci/admin-list.yaml

```diff
@@ -0,0 +1,5 @@
+admin-list:
```

Comment:
added a simple README, please have a look,
currently it is only jenkins that will use the file, it will be used by a jenkins job to update who can trigger CI.